aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -1867,15 +1868,30 @@ std::tuple<Tensor, Tensor, Tensor> _lu_with_info_cuda(const Tensor& self, bool p
   Tensor pivots_tensor = at::arange(1, k + 1, self.options().dtype(at::kInt)).expand(req_size).contiguous();
```

Comment:
Does pivot tensor have to be initialized like this, or would an `at::empty` work? `arange` on CUDA would launch an extra kernel to initialize data.

Reply:
Good point. Actually let me try to test this and find out.

Reply:
Its not necessary to initiliaze it as pointed out here: https://github.com/pytorch/pytorch/pull/56887#discussion_r629531126

Reply:
Actually I was wrong. Have changed it to reflect that empty tensor works for pivoted LU.